src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -1010,22 +1010,13 @@ class InferInput:
     Parameters
     ----------
     name : str
-        The name of input whose data will be described by this object
-    shape : list
-        The shape of the associated input. Default value is None.
-    datatype : str
-        The datatype of the associated input. Default is None.
+        The name of input whose data will be described by this object.
 
     """
 
-    def __init__(self, name, shape=None, datatype=None):
+    def __init__(self, name):
```

Comment:
Why do you remove shape and datatype. Every input must explicitly specify a shape and datatype and the constructor is a natural place to do this. We want to make these required so don't provide a default.